### PR TITLE
IN-214 Add local environment for development purposes

### PR DESCRIPTION
**Pull request by jamesrwarren**
## Purpose

To generate a local environment that can be used by developers to either develop against or for other developers to integrate with. For example, a sirius dev could spin up the local environment and run their tests with their outbound url pointed to it in local development.

## Approach

I wanted something that was very configurable and doesn't require devs to install loads of client tools. As such I went for a model whereby they can spin up the dockerised environment and use endpoints to set the DB in whatever way they like.

## Learning

Dynamodb is interesting and not as simple as transact SQL based databases.

## Checklist

* [x] I have performed a self-review of my own code
* [x] I have added relevant logging with appropriate levels to my code
* [x] I have updated documentation where relevant
* [ ] I have added tests to prove my work
* [ ] The product team have tested these changes

### CODE REVIEW

File: docs/support_files/create_rows.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "rows": [
```
Comment (jmackown) [inline docs/support_files/create_rows.json:2]: These should have a `dob` in each row

File: lambda_functions/v1/functions/lpa_codes/app/api/lets_see_about_this.py [hunk 2]
```diff
@@ -7,9 +7,24 @@
 logger = custom_logger()
 
 
+def aws_credentials():
```
Comment (jmackown) [inline lambda_functions/v1/functions/lpa_codes/app/api/lets_see_about_this.py:10]: Should this be in here? With the fake credentials?

Comment (jamesrwarren) [reply lambda_functions/v1/functions/lpa_codes/app/api/lets_see_about_this.py:10]: You're right. Should be using docker env vars instead. Good call :)

File: docs/support_files/create_rows.json (hunk 1 above)
Comment (jamesrwarren) [reply docs/support_files/create_rows.json:2]: Updated

**Pull request merged**